src/test/java/org/disq_bio/disq/HtsjdkReadsRddTest.java

```diff
@@ -390,4 +440,21 @@ public void testOverwrite() throws IOException, URISyntaxException {
     htsjdkReadsRddStorage.write(htsjdkReadsRdd, outputPath);
     Assert.assertEquals(expectedCount, AnySamTestUtil.countReads(outputPath));
   }
+
+  @Test
+  public void testNoSBIIndexWrittenWhenNotCoordinateSorted() throws Exception {
```

Comment:
Why don't we write these for non-coordinate sorted bams?  I thought they didn't require any particular sorting?

Reply:
Ditto